index.js

```diff
@@ -44,7 +44,7 @@ var cm = require("sdk/context-menu");
 
 cm.Item({
   label: "Send to mini player",
-  context: cm.SelectorContext('[href*="youtube.com"], [href*="youtu.be"]'),
+  context: cm.SelectorContext('[class*="yt-uix-sessionlink"], [href*="youtube.com"], [href*="youtu.be"]'),
```

Comment:
I think that checks for classname and attributes specific to the youtube domain should be in a separate call to `cm.Item`

This would probably look something like:

``` js
cm.Item({
  label: "Send to mini player",
  context: [
    cm.URLContext(['*.youtube.com']),
    cm.SelectorContext('href*="watch"')
  ],
```